src/routes/newsletters/_query.ts

```diff
@@ -38,3 +42,36 @@ export const newslettersQuery = `{
     }
   }
 }`;
+
+/* eslint-disable */

```

Comment:
Why is `eslint` disabled here?

Reply:
Good question. There was likely an eslint error that was not going away easily. Any unnecessary eslint comments (including the one you've mentioned) should be removed before this PR is fully approved/merged.